README.md

```diff
@@ -17,4 +17,215 @@ Aside from these lines, all the other code in this repository has been written b
 
 ## Usage
 
-It's recommended to use the `blend-settings` UI, instead of the `blend` CLI.
\ No newline at end of file
+It's recommended to use the `blend-settings` UI, instead of the `blend` CLI.
+
+dependentcies require for both methods
+
+podman 
+
+## Building Manually
+To build manually, you are going to need the following dependentcies
+
+pacman fakeroot git binutils npm electron(maunaully downloaded from github, see later) bzip2 make gcc
+
+Install the dependentcies from your package manager except for electron
+
+i.e sudo dnf install pacman fakeroot git binutils npm bzip2 make gcc
+
+Clone the repo for building blend
+
+git clone https://github.com/blend-os/blend-pkg
+
+Then download electron from https://github.com/electron/electron/releases/download/v24.2.0/electron-v24.2.0-linux-x64.zip
```

Comment:
```suggestion
Download electron from https://github.com/electron/electron/releases/latest
```
It is recommended to use latest, as the link will get outdated with newer releases.